book/src/csi-node-info-object.md

```diff
@@ -15,7 +15,7 @@ It serves the following purposes:
 2. Driver availability
   * A way for kubelet to communicate to the kube-controller-manager and kubernetes scheduler whether the driver is available (registered) on the node or not.
 3. Volume topology
-  * The CSI `GetNodeInfo` call returns a set of key/value labels identifying the topology of that node. Kubernetes uses this information to to do topology aware provisioning, so it stores the key/values as labels on the Kubernetes node object. In order to recall which `Node` label keys belong to a specific CSI driver, the kubelet stores the keys in the `CSINodeInfo` object for future reference.
+  * The CSI `GetNodeInfo` call returns a set of keys/values labels identifying the topology of that node. Kubernetes uses this information to to do topology-aware provisioning, so it stores the key/values as labels on the Kubernetes node object. In order to recall which `Node` label keys belong to a specific CSI driver, the kubelet stores the keys in the `CSINodeInfo` object for future reference.
```

Comment:
do you want to link https://kubernetes.io/docs/concepts/storage/storage-classes/#volume-binding-mode

Reply:
Done.